### PR TITLE
Docs : Update Readme file and fix several flake8

### PR DESCRIPTION
**Pull request by ghost**
See the Readme @44yu5h @chimosky @Walterbender.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-## RPi Control Center
+# RPi Control Center
+
+A control center application for Raspberry Pi, designed to manage and monitor various Raspberry Pi functionalities through a user-friendly interface.
+
+<img src="screenshots/en/img.png" width="75%" title="rpicc Activity">
+
```
Comment (44yu5h) [inline README.md:8]: Can you please add these 2 as well
https://drive.google.com/file/d/1TjmIqA_NT6UQKrRgRcB3MI0oKlnUIBLy/view?usp=sharing
https://drive.google.com/file/d/1TvzNgIhXcxYd8-CjNZQF2rJ893CpO6Cr/view?usp=sharing
https://drive.google.com/file/d/1Jb5YMoe_ExkrR22nJTc2i_GYF2srHRkS/view?usp=sharing

Thanks

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+  - `RPi.GPIO`
+
+## Installation
+- Install the required libraries: See req.txt files for the required libraries.
+
+## Usage
+- Run the application using the following command:
+  ```
+  python3 activity.py
+  ```
```
Comment (44yu5h) [inline README.md]:
```suggestion
```
They will be run from the home screen of sugar, so I don't think that is necessary

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+## Requirements
+- Raspberry Pi with Raspbian OS
+- Python 3.x
+- Required Python libraries:
+  - `gi`
+  - `psutil`
+  - `RPi.GPIO`
+
+## Installation
+- Install the required libraries: See req.txt files for the required libraries.
```
Comment (44yu5h) [inline README.md]: Maybe we can add steps to the install activity too.
Eg: `cd ~/Activities` `git clone - `

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:8]: Ok Will add them Thanks!!

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Ok so rather giving tut on how to run activity on sugar will be useful like you see example of other sugar activity readme.

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Yes will rather give links to premade tuts provided by sugar on running the activity.

**Pull request closed without merging**